input/images/fhirpath.g4

```diff
@@ -149,8 +149,9 @@ STRING
         ;
 
 // Also allows leading zeroes now (just like CQL and XSD)
+// Also allows the trailing L for Long (STU)
 NUMBER
-        : [0-9]+('.' [0-9]+)?
+        : [0-9]+('.' [0-9]+)? | [0-9]+ 'L'?
```

Comment:
@brianpos @brynrhodes I thought this was going to changed.   As it currently is, doesn't this permit a Long value in a Quantity?  If so, how is that convertible to FHIR Quantity?